packages/appeals-service-api/package.json

```diff
@@ -26,7 +26,7 @@
 		"test": "jest",
 		"test:cov": "jest --coverage --verbose",
 		"test:watch": "jest --watchAll --runInBand",
-		"integration-test": "npm run db:generate && jest --config ./jest.developer-test.config.js"
+		"integration-test": "npm run db:generate && jest --runInBand --config ./jest.developer-test.config.js"
```

Comment:
needed to make integration tests sequential due to using the same db, if this is too slow could update to not clear the db between tests